input/fsh/value_sets.fsh

```diff
@@ -2,129 +2,40 @@
 
 ValueSet:    VaccinationCredentialVaccineValueSet
 Id:          vaccination-credential-vaccine-value-set
-Title:       "Value set for identifying vaccines"
-Description: "This value set includes codes for identifying vaccinations."
+Title:       "Vaccines value set"
+Description: "This value set includes codes for identifying vaccines."
 
+// CVX codes - identifies disease and type of vaccine
 * include codes from system http://hl7.org/fhir/sid/cvx
 
```

Comment:
consider adding
* include codes from system SNOMED_CT where concept descendent-of #787859002

or slices for CVX+MVX, SNOMED, and (when ready) ICD11 with appropriate invariants

Reply:
Thanks! I'm going to prototype what this would look like in a separate PR.